### PR TITLE
Version history for 5.1.3

### PR DESCRIPTION
**Pull request by hflynn**
Will be staged at https://www.openmicroscopy.org/site/support/omero5.1-staging/users/history.html tomorrow.

Final decisions still needed on whether to include ROI preview fixes, gateway...

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+5.1.3 (July 2015)
+-----------------
+
+A bug-fix release. Improvements include:
+
+-  better reporting for the 'delete' and 'chgrp' functionality in the CLI
+-  tagging fixes, including for tagging ome-tiff images at import
+-  fixed display of images in plates with multiple acquisitions
+-  Support for WSGI OMERO.web deployment
+-  Group admins and owners can now change ownership of data via the CLI
```
Comment (mtbc) [inline history.txt]: suddenly we're starting with initial capitals

File: history.txt [hunk 2]
```diff
@@ -1,6 +1,31 @@
...
+-  fixed OMERO.mail service for web errors
+
+Developer updates include:
+
+-  OMERO.web clean-up (removal of '-locked')
+-  refactoring of 'Chown2' and 'Chmod2'
```
Comment (mtbc) [inline history.txt]: Chgrp2 and Delete2 also were refactored since 5.1.2.

**Comment by hflynn** [comment]
Current plan is for separate blog post/announcement about decoupling

**Comment by joshmoore** [comment]
Possible dev items from me:

 * https://github.com/openmicroscopy/openmicroscopy/pull/3943 pointing to the generated HTML. Useful for devs (cc @sbesson )
 * I might need to have a look around for "unicode" head-ups (https://github.com/openmicroscopy/openmicroscopy/pull/3950)
 * https://github.com/openmicroscopy/openmicroscopy/pull/3790 : "Addition of dynamic scripts"

Also:
 * removal of some share UI : https://github.com/openmicroscopy/openmicroscopy/pull/3813

**Comment by joshmoore** [comment]
Merging. Any further suggestions should be opened as separate PRs.

**Pull request merged**